### PR TITLE
add actual and upcoming overload for limiViolations

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/service/StudyService.java [hunk 1]
```diff
@@ -1123,10 +1123,10 @@ public Integer getContingencyCount(UUID studyUuid, List<String> contingencyListN
         return actionsService.getContingencyCount(networkuuid, variantId, contingencyListNames);
     }
 
+    // TO DELETE
```
Comment (FranckLecuyer) [inline src/main/java/org/gridsuite/study/server/service/StudyService.java:1126]: So, remove this method, which is not used anymore

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/study/server/service/StudyService.java (hunk 1 above)
Comment (souissimai) [reply src/main/java/org/gridsuite/study/server/service/StudyService.java]: done

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: a rename requested, if possible

**Review by FranckLecuyer — APPROVED** [review]

File: src/main/java/org/gridsuite/study/server/service/StudyService.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (souissimai) [reply src/main/java/org/gridsuite/study/server/service/StudyService.java]: done

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Pull request merged**